libs/core/src/lib/list/list-item/list-item.component.ts

```diff
@@ -199,6 +200,11 @@ export class ListItemComponent
     /** @hidden */
     readonly _uniqueId = 'fd-list-item-' + ++listItemUniqueId;
 
+    /** @hidden */
+    readonly _list = inject(FD_LIST_UNREAD_INDICATOR, {
```

Comment:
Can you explain what's going on here? Just curious, I don't understand

Reply:
We inject list here to get it's byline state (https://github.com/SAP/fundamental-ngx/pull/9553/files#diff-dc5ece276511bdd17a8ed4a0d3b869f73564d6b6dc759e79d64df46e2edc09b1R8) since unread indicator is available only for the byline list